web/views/cart.hbs

```diff
@@ -0,0 +1,40 @@
+<main class="cart-container">
+  <div class="cart-container--head">
+    <div class="cart-container--head_text">My Cart ({{cart.count}} items)</div>
+  </div>
+
+  {{#each cart.items}}
+  <div class="cart-item" data-id="{{product.id}}">
+    <img src="{{product.imageURL}}" alt="{{product.name}}" class="cart-item__img" />
+    <div class="cart-item-detail">
+      <h1 class="cart-item-detail__heading">{{product.name}}</h1>

```

Comment:
Why is H1 being used inside each product? and why this is not an independent component?